libs/user-repository.ts

```diff
@@ -5,13 +5,18 @@
 import * as fs from 'fs'
 import database from './db.json'
 import { UserPropDefault } from './../models/user.model'
-import type { IUser, LoginTypes } from './../models/user.model'
+import type {
+  IUser,
+  LoginTypes,
+  UserUpdateFormTypes
+} from './../models/user.model'
 
 export const userRepository = {
-  getAll: () => database.users,
+  getAll: () => database.users, // array users list
```

Comment:
```suggestion
    getAll: () => database.users as IUser[], // array users list
```

just define return type of the function, no need to cast `as IUser[]` somewhere else